master/buildbot/steps/source/p4.py

```diff
@@ -231,12 +232,18 @@ def _buildVCCommand(self, doCommand):
         command.extend(doCommand)
 
         def encodeArg(arg):
-            if isinstance(arg, string_types):
+            if isinstance(arg, binary_type):
```

Comment:
I think we should have an utility for this kind of thing in util/__init__.py near ascii2unicode we good have enforce_utf8_string()